src/main/kotlin/Composite/Main.kt

```diff
@@ -0,0 +1,19 @@
+package Composite
+
+import Composite.base.ChineseCooker
+import Composite.base.ItalianCooker
+import Composite.base.Kitchen
+
+fun main(args : Array<String>) {
```

Comment:
Stop using Main, We can see how it used in Test! 

That you are trying to test in your main, test it in unit tests!!